src/Middleware/LegacyMiddleware.php

```diff
@@ -62,7 +64,7 @@ public function process(
             /** @var Translator $translator */
             $translator = $this->container->get('translator');
 
-            $page = 404;
+            $page = '404';
```

Comment:
This is just a "workaround" and setting strict typing breaks the beautiful custom error pages like 404 :(

```suggestion
            $page = 404;
```

Reply:
The `renderPage(string $page,` method should be `renderPage(mixed $page,`

Reply:
uff